gdrive

```diff
@@ -186,32 +193,40 @@ def upload(service, filepath):
 
 def read_argv():
     """Read arguments"""
-    list_all = file_id = file_name = None
+    file_id = file_name = list_all = download_last = None
```

Comment:
You're using "file_name" and "filename" in your code. Keep the consistency and use the same nomenclature. I'd suggest "filename".

Reply:
Renamed every occurrence of 'file_name' to 'filename'.